t/jv-boolean.t

```diff
@@ -0,0 +1,31 @@
+use lib '.';
+use t::Helper;
+use Test::More;
+use utf8;
+use JSON;
```

Comment:
Please don't depend on "JSON.pm" (since it's not a core module). Do this instead:

```
use lib;
use t::Helper; # <-- this imports Test::More and utf8
```